[KOGITO-2118] - Write script to retrieve latest snapshot urls for services jar artifacts (#154)
kiegroup/kogito-images: scripts/README.md

It will look for the current RC images available on [quay.io](https://quay.io/organization/kiegroup) to increase the rc tag 
accordingly then push the new tag so it can be tested by others. 

#### Updating Kogito Images service artifacts.

The update-artifacts script will help in fetching the URL of latest artifacts from the repo
and update them in the `module.yaml` files of the kogito services

#### Script dependencies

The `update-artifacts.py` has some dependencies that needs to be manually installed:

```bash
$ pip install -U ruamel.yaml
$ pip install -U elementpath
```

> It's usage is pretty simple as well:

Reviewer: Grammatical Error: 
*Its not It's

It will look for the current RC images available on [quay.io](https://quay.io/organization/kiegroup) to increase the rc tag 
accordingly then push the new tag so it can be tested by others. 

#### Updating Kogito Images service artifacts.

The update-artifacts script will help in fetching the URL of latest artifacts from the repo
and update them in the `module.yaml` files of the kogito services

#### Script dependencies

The `update-artifacts.py` has some dependencies that needs to be manually installed:

```bash
$ pip install -U ruamel.yaml
$ pip install -U elementpath
```
It's usage is pretty simple as well:

```bash
$ python update-artifacts.py
```

> This script also accepts `--snapshot-version` as argument in specifying the current version of the repository
> from which the artifacts needs to be fetched

Reviewer 2: ```suggestion
This script also accepts `--snapshot-version` as argument in specifying the current version of artifacts which need to be fetched
```

```bash
$ python update-artifacts.py --snapshot-version='8.0.0-SNAPSHOT'
```

> if the argument is not passed it takes the default value of `8.0.0-SNAPSHOT`

Reviewer 2: ```suggestion
if no argument is given, it takes the default value of `8.0.0-SNAPSHOT`
```
